### ISSUE TITLE
.jobsets not adding new PRs to jobsets list

### ISSUE THREAD

**Issue by alastairong** [Holo-Host/holo-nixpkgs#745]
`.jobsets` checks for new PRs regularly (every few seconds), then it’s supposed to build `jobsets.json` file and then Hydra (somehow?) picks up new jobsets from it and starts building them. This workflow works fine on Hydra 19.09, but not on 20.09, where the build of jobsets.json completes (and the file is verified to be up-to-date), but Hydra doesn’t register the change.

**Comment by peeech** [comment]
I was able to observe an issue in details:
- on `Mar 22 20:20:00` new [PR#752](https://github.com/Holo-Host/holo-nixpkgs/pull/752/commits/aa7f0f8af912523c4fe179236582838c9afa6cee) has been created
- on `Mar 22 20:21:11` [Production Hydra](https://hydra.holo.host/project/holo-nixpkgs) polled github, evaluated new `.jobset` and started evaluating new PR#752:
```
Mar 22 20:21:11 hydra hydra-notify[11316]: GithubStatus_Debug job name holo-nixpkgs:.jobsets:jobsets
Mar 22 20:22:54 hydra hydra-notify[11316]: GithubStatus_Debug job name holo-nixpkgs:752:hpos-admin-api.aarch64-linux
Mar 22 20:22:54 hydra hydra-notify[11316]: GithubStatus_Debug job name holo-nixpkgs:752:hpos-admin-api.x86_64-linux
Mar 22 20:22:54 hydra hydra-notify[11316]: GithubStatus_Debug job name holo-nixpkgs:752:hpos-holochain-api.x86_64-darwin
```
- since then PR#752 is evaluated and available via UI on [Production Hydra](https://hydra.holo.host/project/holo-nixpkgs)
- on `Mar 22 20:20:56` [Hydra-dev](https://hydra-dev.holo.host/project/holo-nixpkgs) made an attempt to evaluate `.jobset`. Log shows following error:
```
Mar 22 20:20:56 hydra-dev hydra-notify[861]: GithubStatus_Debug job name holo-nixpkgs:.jobsets:jobsets
Mar 22 20:20:56 hydra-dev hydra-notify[861]: GithubStatus_Debug job name holo-nixpkgs:.jobsets:jobsets
Mar 22 20:20:57 hydra-dev hydra-notify[27498]: warning: unknown setting 'extra-sandbox-paths'
Mar 22 20:20:57 hydra-dev hydra-notify[27498]: warning: unknown setting 'gc-check-reachability'
Mar 22 20:20:57 hydra-dev hydra-notify[27498]: error: --- InvalidPath --- nix
Mar 22 20:20:57 hydra-dev hydra-notify[27498]: path '/nix/store/yi9yxyncqcf8scdwaqqr8wr07xiim4wi-jobsets.json' is not a valid store path
Mar 22 20:20:57 hydra-dev hydra-notify[861]: ERROR: failed to readNixFile /nix/store/yi9yxyncqcf8scdwaqqr8wr07xiim4wi-jobsets.json: command `nix --experimental-features nix-command cat-store --store http>
Mar 22 20:20:57 hydra-dev hydra-notify[861]: GithubStatus_Debug job name holo-nixpkgs:.jobsets:jobsets
Mar 22 20:20:57 hydra-dev hydra-notify[861]: GithubStatus_Debug job name holo-nixpkgs:.jobsets:jobsets
```
- at the moment of writing (`Mar 23 08:45:00`) PR#752 is not visible in [Hydra-dev](https://hydra-dev.holo.host/project/holo-nixpkgs) UI

**Comment by peeech** [comment]
fixed in #780
